openmdao/docs/usr-guide/tutorials/paraboloid-tutorial.rst

```diff
@@ -257,6 +257,42 @@ The output should look like this:
 
    -15.0
 
+The `IndepVarComp` component is used to define a source for a parameter that
+we want to use as a design variable for a driver. In our case, we want to
```

Comment:
might want to use the phrase "independent variable" here, to be consistent with the component name  (or change the component name to DesignVarComp ;))